envs/example/defaults.yml

```diff
@@ -322,15 +323,15 @@ keystone:
     - name: heat
       type: orchestration
       description: 'Heat Orchestration API'
-      public_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      internal_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      admin_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
+      public_url: https://{{ endpoints.main }}:8004/v1/%(tenant_id)s
```

Comment:
If we change the heat port numbers here we will break anybody that upgrades from `RELEASE-1.2.x` that rely on the existing port numbers.

Heat is not a new component and we need to consider the upgrade strategy required to ensure that we don't adversely affect existing users.   

Our agreed upon upgrade strategy was to solve the port remapping in #1144 in a way that makes it easy to preserve port numbers for existing stacks while allowing new port numbers for new stacks.